docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -1,5 +1,6 @@
 package com.nirima.jenkins.plugins.docker;
 
+import com.github.dockerjava.api.model.*;
```

Comment:
That mean that there are enough imports for collapsing them. It doesn't influence runtime performance.